gnssanalysis/gn_io/igslog.py

```diff
@@ -138,22 +146,18 @@ def find_recent_logs(logs_glob_path: str, rnx_glob_path: str = None) -> _pd.Data
     return recent_logs_df
 
 
-_REGEX_VERSION_1 = _re.compile(rb"""(site log\))""")
-_REGEX_VERSION_2 = _re.compile(rb"""(site log v2)""")
-
-
 def determine_log_version(data: bytes) -> str:
     """Given the byes object that results from reading an IGS log file, determine the version ("v1.0" or "v2.0")
 
     :param bytes data: IGS log file bytes object to determine the version of
     :return str: Return the version number: "v1.0" or "v2.0" (or "Unknown" if file does not conform to standard)
     """
 
-    result_v1 = _REGEX_VERSION_1.search(data)
+    result_v1 = _REGEX_LOG_VERSION_1.search(data)
     if result_v1:
         return "v1.0"
 
```

Comment:
For efficiency and robustness, I'd suggest trimming the binary string to the first line, before running the regex over it.

```suggestion
    first_line_bytes = data.split(b"\n")[0]
    result_v1 = _REGEX_LOG_VERSION_1.search(first_line_bytes)
    if result_v1:
        return "v1.0"
```